Android/GoogleDagger/source/Dagger/Additions/Additions.cs

```diff
@@ -0,0 +1,5 @@
+﻿using System;
+
+namespace Google.Dagger.Dagger {
```

Comment:
Namespace change here, though in practice we could just remove this file altogether if it's not specifying anything